### PR TITLE
[FIX](map)fix map proto contains_null

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes
when we select map in order by and limit;  be node will coredump
here is stack info 
截屏2023-07-21 23 55 54

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by amorynan** [comment]
run buildall

**Review by eldenmoon — APPROVED** [review]
LGTM

File: gensrc/proto/types.proto [hunk 1]
```diff
@@ -51,6 +51,8 @@ message PTypeNode {
     repeated PStructField struct_fields = 3;
     // only used for complex types, such as array, map and etc.
     optional bool contains_null = 4;
+    // update for map/struct type
+    repeated bool contains_nulls = 5;
```
Comment (xiaokang) [inline gensrc/proto/types.proto:55]: what's the difference to contains_null?

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 47.03 seconds
 stream load tsv:          506 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          29.7 seconds inserted 10000000 Rows, about 336K ops/s
 storage size: 17169166287 Bytes

File: gensrc/proto/types.proto (hunk 1 above)
Comment (amorynan) [reply gensrc/proto/types.proto:55]: contains_nulls is repeated which is vector for null , just keep same with contains null in thrift

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**